### PR TITLE
Fix(UI): Fix pending services dot display

### PR DESCRIPTION
**Pull request by Thebarda**
## Description

This uses the Badge component from Material UI to display the "pending" dot

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 20.04.x
- [ ] 20.10.x
- [ ] 21.04.x
- [x] 21.10.x (master)

<h2> How this pull request can be tested ? </h2>

- Create a service/host with a "Pending" status
-  The services/hosts top counter should display the "pending" dot in the right way

## Checklist

- [x] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

**Review by bdauria — APPROVED** [review]

File: package.json
Comment (jdelpierre) [inline package.json:90]: Are you going to wait until it is merged into centreon-frontend or will you have to think about doing it later?

**Review by jdelpierre — APPROVED** [review]

**Pull request merged**
